packages/runner-shared/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@packages/runner-shared",
+  "version": "0.0.0-development",
+  "private": true,
+  "main": "src/index.ts",
+  "scripts": {
+    "test": "yarn test-unit",
+    "test-unit": "mocha --config test/.mocharc.json src/**/*.spec.* --exit"
```

Comment:
nit: Could we have used Cypress Component Testing for that ?

Reply:
Maybe one day, but there is already 200+ tests written with enzyme, good to keep them for now.